packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -120,7 +132,30 @@ const setupTokenListController = (
   });
 
   return { tokenList, tokenListMessenger };
-};
+}
+
+/**
+ * Builds a messenger that `TokenDetectionController` can use to communicate with other controllers.
+ * @param controllerMessenger - The main controller messenger.
+ * @returns The restricted messenger.
+ */
+function buildTokenDetectionControllerMessenger(
+  controllerMessenger?: MainControllerMessenger,
+): TokenDetectionControllerMessenger {
+  return (controllerMessenger ?? getControllerMessenger()).getRestricted({
```

Comment:
Nit: Can we use the default argument syntax here?

``` suggestion
  controllerMessenger: MainControllerMessenger = getControllerMessenger(),
): TokenDetectionControllerMessenger {
  return controllerMessenger.getRestricted({
```

Reply:
Fixed here: https://github.com/MetaMask/core/pull/3609/commits/1337d199b42db63cc94fa0a62cd645c5535a30d8